helm/prometheus-rules/templates/alerting-rules/disk.workload-cluster.rules.yml

```diff
@@ -21,7 +21,13 @@ spec:
         area: kaas
         cancel_if_outside_working_hours: "true"
         severity: page
+        {{- if eq .Values.managementCluster.provider.kind "aws" }}
```

Comment:
Why not use the same syntax as for rocket? i.e. `{{- else if or (eq .Values.managementCluster.provider.kind "aws") (eq .Values.managementCluster.provider.kind "azure") }}`

Reply:
yeah let's do that. Take a look now.